assets/js/main.js

```diff
@@ -469,6 +469,11 @@ class Carousel {
             var left = 50 + 100.0 / carousel.itemsRow * idxDiff;
             // Allows image overlap with each other.
             var width = carousel.overlapFactor / carousel.itemsRow;
+
+            if (idx >= 4) {
```

Comment:
Instead of hardcode idx>=4 here, how about use $(item).attr to retrieve some DOM attr  and set its width?